### PR TITLE
Replace Github -> GitHub (Spanish)

### PR DESCRIPTION
**Pull request by lipis**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by lipis** [comment]
After we merge all those.. I'll do a final check on the whole project again :)

File: guide/spanish/react/what-are-react-props/index.md
Comment (lipis) [inline guide/spanish/react/what-are-react-props/index.md]: Still not sure what is going on here.. but the result looks legit! Some hidden character..

**Comment by RandellDawson** [comment]
@lynxlynxlynx It would be very helpful if you could review the following PRs which make changes to one or more of the same files on this PR.  Once all of those have been closed/merged, then we can resolve any merge conflicts that result on this PR.  We would really like to get this PR merged quickly since it affects so many files (43).  Thanks in advance.

https://github.com/freeCodeCamp/freeCodeCamp/pull/20857
https://github.com/freeCodeCamp/freeCodeCamp/pull/25138
https://github.com/freeCodeCamp/freeCodeCamp/pull/26280
https://github.com/freeCodeCamp/freeCodeCamp/pull/26477
https://github.com/freeCodeCamp/freeCodeCamp/pull/26611
https://github.com/freeCodeCamp/freeCodeCamp/pull/30207
https://github.com/freeCodeCamp/freeCodeCamp/pull/30715
https://github.com/freeCodeCamp/freeCodeCamp/pull/31763
https://github.com/freeCodeCamp/freeCodeCamp/pull/31778
https://github.com/freeCodeCamp/freeCodeCamp/pull/31927
https://github.com/freeCodeCamp/freeCodeCamp/pull/34099

**Comment by lynxlynxlynx** [comment]
Most are done, one requiring further work from the author and 3 being the ones touching how-to-setup-freecodecamp-locally.md: #31927 #30715 #25138 . Those need someone with a better command of the language.

**Comment by RandellDawson** [comment]
@lynxlynxlynx Thanks for the fast turn-around for these.  Also, whoever reviews those 3 how-to-setup-freecodecamp-locally.md should validate the English `master` version has not changed enough to make any of the changes obsolete.

**Comment by RandellDawson** [comment]
@lipis Don't worry about any merge conflicts until the other 4 PRs we are waiting on have been closed/merged.  I want to take care of any merge conflicts.

File: guide/spanish/react/what-are-react-props/index.md
Comment (RandellDawson) [reply guide/spanish/react/what-are-react-props/index.md]: @lipis Getting close to merging this one.  What is the issue you are concerned about here?

Comment (lynxlynxlynx) [reply guide/spanish/react/what-are-react-props/index.md]: React shouldn't be translated. TypeChecking is bad already in the original.

Comment (RandellDawson) [reply guide/spanish/react/what-are-react-props/index.md]: @lynxlynxlynx If you look at the changes, it does look like what shows above.  I think this is a non-issue.

**Comment by RandellDawson** [comment]
@raisedadead There are still 3 outstanding earlier PRs (#31927 #30715 #25138 ) which have changes to one or more of the same files as this PR.  They all already have merge conflicts and many of the documents' English version have changed significantly since these PRs have been created, so the translations may not even be relevant anymore.  My question is, can we go ahead merge this PR?

**Comment by raisedadead** [comment]
Yes please.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Pull request merged**
